src/ImageItPanel.tsx

```diff
@@ -96,12 +96,17 @@ export const ImageItPanel: React.FC<Props> = ({
             // Get mapping by id || undefined
             const mapping = sensor.mappingId ? mappings.find(mapping => sensor.mappingId === mapping.id) : undefined;
 
+            const renderedLink = replaceVariables(sensor.link);
+            const rendName = replaceVariables(sensor.name);
+
```

Comment:
Yea it would be difficult in Sensor.tsx, but I think it's possible to do the whole process in 2 lines here

```suggestion
            sensor.link = replaceVariables(sensor.link);
            sensor.name = replaceVariables(sensor.name);

```

Reply:
Tested with your modfications - it works , BUT these direct modifications of sensor.link and sensor.name in the code also changes corresponding values in the Edit panel (i.e. when I open edit panel in Grafana I see in the Name field (for corresponding sensor) value without variable).
I.e. during initial creation of dashnoard I specified for sensor in Name field value "${location}1", but after refreshing this value changed to "KB02-1" (value of $location dashboard variable was "KB02-")

I think it is because sensor object here corresponds to persistent object which contains configuration parameters during panel editing and changing member variables in the code for this object is bad idea...

Reply:
Thanks for checking!